manifests/10_operator.yaml

```diff
@@ -57,6 +57,10 @@ spec:
                   fieldPath: metadata.namespace
             - name: OPERATOR_NAME
               value: "marketplace-operator"
+            - name: POD_NAME
```

Comment:
Please add these changes to the deployments found in `/manifests`,`/deploy`, and `/deploy/upstream`.